### DISCUSSION TITLE
Make it so the program disables itself when you're in a fullscreen app

### DISCUSSION THREAD

**Discussion by RarioTrarioWario** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#24] [Ideas]
Make it so the program disables itself when you're in a fullscreen app because sometimes when I am in a fullscreen game for example it activates randomly minimizing the app.

**Comment by HerMajestyDrMona** [comment]
What fullscreen app does it happen in?

The program does not activate if the mouse pointer is not under the visible taskbar or notification area window: 
https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/blob/54eec91a1ff4757fb60fbcaf11654755660b4690/Windows11DragAndDropToTaskbarFix/Windows11DragAndDropToTaskbarFix.cpp#L2077

Therefore this issue should not exist. It is this way since version 1.0.

How are you so sure that it's Windows11DragAndDropToTaskbarFix minimizing your fullscreen apps, and not some different running software? Does it happen right after the mouse is in the supposed taskbar area, or what?

Please provide some more details regarding it, because it's more like an Issue, not a feature request, since it's already working the way you wanted it to work.

**Reply by RarioTrarioWario** [reply]
Nevermind, I fixed it, it had nothing to do with the drag & drop fix.

**Reply by HerMajestyDrMona** [reply]
Yes, as I expected. Would you please share what was causing the problem (so other people who are potentially reading it will be able to find help)?
